tests/tlsctx.c

```diff
@@ -33,6 +37,161 @@ static void ossl_err_print(void)
     }
 }
 
+static EVP_PKEY *load_key(const char *uri)
```

Comment:
I see this is taken from the `tests/tcmpkeys.c`. Would it make sense to move it to some helpers file to avoid a need to copy&paste it in different tests? The code is mostly obvious, but it has already a workaround for OpenSSL <3.2 but we might need more in the future, which would make it much harder to maintain.

Reply:
Ok, splitting off a util.c file